package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Geospatial data visualization platform.",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">= 16.0.0"
+    "node": ">= 20.0.0"
```

Comment:
Should the `types/node` package version be updated to 20.x?

Reply:
That follows the typescript version. Will do it when merging https://github.com/TerriaJS/terriajs/pull/7342